data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -460,7 +547,7 @@ class ApiServerClient {
     }
     return Promise.try(() => this.init())
       .then(() => apiserver.apis[opts.resourceGroup][CONST.APISERVER.API_VERSION]
-        .namespaces(CONST.APISERVER.NAMESPACE)[opts.resourceType].get(query))
+        .namespaces(CONST.APISERVER.DEFAULT_NAMESPACE)[opts.resourceType].get(query))
```

Comment:
why is it default name space here? Cant you get all resources belonging to a specific namespace if thats whats provided in opts?

Reply:
@amitmalav can you check this?

Reply:
We can. But currently no use case for getting resources from namespaces other than `default` as this method is only being used in Bosh-Operator currently. Hence did not make change. We can enhance this method whenever some use case comes. WDYT?

Reply:
I guess its better if you use the namespace as in put in opts or default it if its undefined. We dont have to revisit this class when such a need arises as we are now supporting namespaces better that this framework class has complete support.

Reply:
Done.